rclcpp/src/rclcpp/graph_listener.cpp

```diff
@@ -48,7 +48,7 @@ GraphListener::GraphListener(const std::shared_ptr<Context> & parent_context)
 
 GraphListener::~GraphListener()
 {
-  this->shutdown(std::nothrow);
+  GraphListener::shutdown(std::nothrow);
```

Comment:
should this class have a note in the header file similar to the other ones?

Reply:
Good point, I updated the docstrings.